benchmark.c

```diff
@@ -17,8 +17,8 @@
 #include <pthread.h>
 #include <time.h>
 
-#include "include/module.h"
-#include "include/device.h"
+#include "module.h"
+#include "device.h"
 
 // #define USE_LEGACY_RANDOM
```

Comment:
Comment (Line 23) still being leaved

Reply:
This will handle in the near future. Before adding this feature to Makefile, I need to leave this comment now. Thanks for reminding this 🙇